package.json

```diff
@@ -15,14 +15,16 @@
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "rollup --config"
+    "build": "rollup --config",
```

Comment:
For now, adding the `--dry-run` flag so that we can ensure this functionality works without truly publishing a new version of the package.